### ISSUE TITLE
[BUG/ISSUE] make install does not move binary from bin to run directory in GCClassic 13.0.0

### ISSUE THREAD

**Issue by ltmurray** [geoschem/geos-chem#649]
I've tested on multiple systems (RHEL; Ubuntu; MacOS), and after calling "cmake ../CodeDir" and "make" in a new GCClassic run/build directory generated by the new 13.0.0 bash script, "make install" does not exist in the Makefile that was generated by cmake. Therefore, it does not copy bin/gcclassic into the run directory. "make install" does work when generating HEMCO standalone after generating a standalone run directory in the src/HEMCO/run code. 

In the short term, a user can just manually mv build/bin/gcclassic to the run directory after running make, but just wanted to point out the issue. I've pasted a copy of a Makefile generated by cmake in the build directory for a standard fullchem run.

```
# CMAKE generated file: DO NOT EDIT!
# Generated by "Unix Makefiles" Generator, CMake Version 3.19

# Default target executed when no arguments are given to make.
default_target: all

.PHONY : default_target

# Allow only one "make -f Makefile2" at a time, but pass parallelism.
.NOTPARALLEL:


#=============================================================================
# Special targets provided by cmake.

# Disable implicit rules so canonical targets will work.
.SUFFIXES:


# Disable VCS-based implicit rules.
% : %,v


# Disable VCS-based implicit rules.
% : RCS/%


# Disable VCS-based implicit rules.
% : RCS/%,v


# Disable VCS-based implicit rules.
% : SCCS/s.%


# Disable VCS-based implicit rules.
% : s.%


.SUFFIXES: .hpux_make_needs_suffix_list


# Command-line flag to silence nested $(MAKE).
$(VERBOSE)MAKESILENT = -s

#Suppress display of executed commands.
$(VERBOSE).SILENT:

# A target that is always out of date.
cmake_force:

.PHONY : cmake_force

#=============================================================================
# Set environment variables for the build.

# The shell in which to execute make rules.
SHELL = /bin/sh

# The CMake executable.
CMAKE_COMMAND = /opt/cmake/bin/cmake

# The command to remove a file.
RM = /opt/cmake/bin/cmake -E rm -f

# Escaping for special characters.
EQUALS = =

# The top-level source directory on which CMake was run.
CMAKE_SOURCE_DIR = /home/ltmurray/test/gc_merra2_fullchem/CodeDir

# The top-level build directory on which CMake was run.
CMAKE_BINARY_DIR = /home/ltmurray/test/gc_merra2_fullchem/build

#=============================================================================
# Targets provided globally by CMake.

# Special rule for the target rebuild_cache
rebuild_cache:
	@$(CMAKE_COMMAND) -E cmake_echo_color --switch=$(COLOR) --cyan "Running CMake to regenerate build system..."
	/opt/cmake/bin/cmake --regenerate-during-build -S$(CMAKE_SOURCE_DIR) -B$(CMAKE_BINARY_DIR)
.PHONY : rebuild_cache

# Special rule for the target rebuild_cache
rebuild_cache/fast: rebuild_cache

.PHONY : rebuild_cache/fast

# Special rule for the target edit_cache
edit_cache:
	@$(CMAKE_COMMAND) -E cmake_echo_color --switch=$(COLOR) --cyan "Running CMake cache editor..."
	/opt/cmake/bin/ccmake -S$(CMAKE_SOURCE_DIR) -B$(CMAKE_BINARY_DIR)
.PHONY : edit_cache

# Special rule for the target edit_cache
edit_cache/fast: edit_cache

.PHONY : edit_cache/fast

# The main all target
all: cmake_check_build_system
	$(CMAKE_COMMAND) -E cmake_progress_start /home/ltmurray/test/gc_merra2_fullchem/build/CMakeFiles /home/ltmurray/test/gc_merra2_fullchem/build//CMakeFiles/progress.marks
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 all
	$(CMAKE_COMMAND) -E cmake_progress_start /home/ltmurray/test/gc_merra2_fullchem/build/CMakeFiles 0
.PHONY : all

# The main clean target
clean:
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 clean
.PHONY : clean

# The main clean target
clean/fast: clean

.PHONY : clean/fast

# Prepare targets for installation.
preinstall: all
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 preinstall
.PHONY : preinstall

# Prepare targets for installation.
preinstall/fast:
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 preinstall
.PHONY : preinstall/fast

# clear depends
depend:
	$(CMAKE_COMMAND) -S$(CMAKE_SOURCE_DIR) -B$(CMAKE_BINARY_DIR) --check-build-system CMakeFiles/Makefile.cmake 1
.PHONY : depend

#=============================================================================
# Target rules for targets named gcclassic

# Build rule for target.
gcclassic: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 gcclassic
.PHONY : gcclassic

# fast build rule for target.
gcclassic/fast:
	$(MAKE) $(MAKESILENT) -f src/CMakeFiles/gcclassic.dir/build.make src/CMakeFiles/gcclassic.dir/build
.PHONY : gcclassic/fast

#=============================================================================
# Target rules for targets named HCO

# Build rule for target.
HCO: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 HCO
.PHONY : HCO

# fast build rule for target.
HCO/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Core/CMakeFiles/HCO.dir/build.make src/HEMCO/src/Core/CMakeFiles/HCO.dir/build
.PHONY : HCO/fast

#=============================================================================
# Target rules for targets named HCOX

# Build rule for target.
HCOX: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 HCOX
.PHONY : HCOX

# fast build rule for target.
HCOX/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Extensions/CMakeFiles/HCOX.dir/build.make src/HEMCO/src/Extensions/CMakeFiles/HCOX.dir/build
.PHONY : HCOX/fast

#=============================================================================
# Target rules for targets named HCOI_Shared

# Build rule for target.
HCOI_Shared: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 HCOI_Shared
.PHONY : HCOI_Shared

# fast build rule for target.
HCOI_Shared/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Interfaces/Shared/CMakeFiles/HCOI_Shared.dir/build.make src/HEMCO/src/Interfaces/Shared/CMakeFiles/HCOI_Shared.dir/build
.PHONY : HCOI_Shared/fast

#=============================================================================
# Target rules for targets named HCOI_Standalone

# Build rule for target.
HCOI_Standalone: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 HCOI_Standalone
.PHONY : HCOI_Standalone

# fast build rule for target.
HCOI_Standalone/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Interfaces/Standalone/CMakeFiles/HCOI_Standalone.dir/build.make src/HEMCO/src/Interfaces/Standalone/CMakeFiles/HCOI_Standalone.dir/build
.PHONY : HCOI_Standalone/fast

#=============================================================================
# Target rules for targets named HeadersHco

# Build rule for target.
HeadersHco: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 HeadersHco
.PHONY : HeadersHco

# fast build rule for target.
HeadersHco/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Shared/Headers/CMakeFiles/HeadersHco.dir/build.make src/HEMCO/src/Shared/Headers/CMakeFiles/HeadersHco.dir/build
.PHONY : HeadersHco/fast

#=============================================================================
# Target rules for targets named GeosUtilHco

# Build rule for target.
GeosUtilHco: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 GeosUtilHco
.PHONY : GeosUtilHco

# fast build rule for target.
GeosUtilHco/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Shared/GeosUtil/CMakeFiles/GeosUtilHco.dir/build.make src/HEMCO/src/Shared/GeosUtil/CMakeFiles/GeosUtilHco.dir/build
.PHONY : GeosUtilHco/fast

#=============================================================================
# Target rules for targets named JulDayHco

# Build rule for target.
JulDayHco: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 JulDayHco
.PHONY : JulDayHco

# fast build rule for target.
JulDayHco/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Shared/GeosUtil/CMakeFiles/JulDayHco.dir/build.make src/HEMCO/src/Shared/GeosUtil/CMakeFiles/JulDayHco.dir/build
.PHONY : JulDayHco/fast

#=============================================================================
# Target rules for targets named NcdfUtilHco

# Build rule for target.
NcdfUtilHco: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 NcdfUtilHco
.PHONY : NcdfUtilHco

# fast build rule for target.
NcdfUtilHco/fast:
	$(MAKE) $(MAKESILENT) -f src/HEMCO/src/Shared/NcdfUtil/CMakeFiles/NcdfUtilHco.dir/build.make src/HEMCO/src/Shared/NcdfUtil/CMakeFiles/NcdfUtilHco.dir/build
.PHONY : NcdfUtilHco/fast

#=============================================================================
# Target rules for targets named KPP_fullchem

# Build rule for target.
KPP_fullchem: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 KPP_fullchem
.PHONY : KPP_fullchem

# fast build rule for target.
KPP_fullchem/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/KPP/fullchem/CMakeFiles/KPP_fullchem.dir/build.make src/GEOS-Chem/KPP/fullchem/CMakeFiles/KPP_fullchem.dir/build
.PHONY : KPP_fullchem/fast

#=============================================================================
# Target rules for targets named KPPFirstPass_fullchem

# Build rule for target.
KPPFirstPass_fullchem: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 KPPFirstPass_fullchem
.PHONY : KPPFirstPass_fullchem

# fast build rule for target.
KPPFirstPass_fullchem/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/KPP/fullchem/CMakeFiles/KPPFirstPass_fullchem.dir/build.make src/GEOS-Chem/KPP/fullchem/CMakeFiles/KPPFirstPass_fullchem.dir/build
.PHONY : KPPFirstPass_fullchem/fast

#=============================================================================
# Target rules for targets named Headers

# Build rule for target.
Headers: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 Headers
.PHONY : Headers

# fast build rule for target.
Headers/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/Headers/CMakeFiles/Headers.dir/build.make src/GEOS-Chem/Headers/CMakeFiles/Headers.dir/build
.PHONY : Headers/fast

#=============================================================================
# Target rules for targets named GeosUtil

# Build rule for target.
GeosUtil: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 GeosUtil
.PHONY : GeosUtil

# fast build rule for target.
GeosUtil/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/GeosUtil/CMakeFiles/GeosUtil.dir/build.make src/GEOS-Chem/GeosUtil/CMakeFiles/GeosUtil.dir/build
.PHONY : GeosUtil/fast

#=============================================================================
# Target rules for targets named JulDay

# Build rule for target.
JulDay: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 JulDay
.PHONY : JulDay

# fast build rule for target.
JulDay/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/GeosUtil/CMakeFiles/JulDay.dir/build.make src/GEOS-Chem/GeosUtil/CMakeFiles/JulDay.dir/build
.PHONY : JulDay/fast

#=============================================================================
# Target rules for targets named NcdfUtil

# Build rule for target.
NcdfUtil: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 NcdfUtil
.PHONY : NcdfUtil

# fast build rule for target.
NcdfUtil/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/NcdfUtil/CMakeFiles/NcdfUtil.dir/build.make src/GEOS-Chem/NcdfUtil/CMakeFiles/NcdfUtil.dir/build
.PHONY : NcdfUtil/fast

#=============================================================================
# Target rules for targets named History

# Build rule for target.
History: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 History
.PHONY : History

# fast build rule for target.
History/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/History/CMakeFiles/History.dir/build.make src/GEOS-Chem/History/CMakeFiles/History.dir/build
.PHONY : History/fast

#=============================================================================
# Target rules for targets named ObsPack

# Build rule for target.
ObsPack: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 ObsPack
.PHONY : ObsPack

# fast build rule for target.
ObsPack/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/ObsPack/CMakeFiles/ObsPack.dir/build.make src/GEOS-Chem/ObsPack/CMakeFiles/ObsPack.dir/build
.PHONY : ObsPack/fast

#=============================================================================
# Target rules for targets named APM

# Build rule for target.
APM: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 APM
.PHONY : APM

# fast build rule for target.
APM/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/APM/CMakeFiles/APM.dir/build.make src/GEOS-Chem/APM/CMakeFiles/APM.dir/build
.PHONY : APM/fast

#=============================================================================
# Target rules for targets named Isorropia

# Build rule for target.
Isorropia: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 Isorropia
.PHONY : Isorropia

# fast build rule for target.
Isorropia/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/ISORROPIA/CMakeFiles/Isorropia.dir/build.make src/GEOS-Chem/ISORROPIA/CMakeFiles/Isorropia.dir/build
.PHONY : Isorropia/fast

#=============================================================================
# Target rules for targets named GeosRad

# Build rule for target.
GeosRad: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 GeosRad
.PHONY : GeosRad

# fast build rule for target.
GeosRad/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/GeosRad/CMakeFiles/GeosRad.dir/build.make src/GEOS-Chem/GeosRad/CMakeFiles/GeosRad.dir/build
.PHONY : GeosRad/fast

#=============================================================================
# Target rules for targets named Hg

# Build rule for target.
Hg: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 Hg
.PHONY : Hg

# fast build rule for target.
Hg/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/GTMM/CMakeFiles/Hg.dir/build.make src/GEOS-Chem/GTMM/CMakeFiles/Hg.dir/build
.PHONY : Hg/fast

#=============================================================================
# Target rules for targets named GeosCore

# Build rule for target.
GeosCore: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 GeosCore
.PHONY : GeosCore

# fast build rule for target.
GeosCore/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/GeosCore/CMakeFiles/GeosCore.dir/build.make src/GEOS-Chem/GeosCore/CMakeFiles/GeosCore.dir/build
.PHONY : GeosCore/fast

#=============================================================================
# Target rules for targets named Transport

# Build rule for target.
Transport: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 Transport
.PHONY : Transport

# fast build rule for target.
Transport/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/GeosCore/CMakeFiles/Transport.dir/build.make src/GEOS-Chem/GeosCore/CMakeFiles/Transport.dir/build
.PHONY : Transport/fast

#=============================================================================
# Target rules for targets named GCClassic

# Build rule for target.
GCClassic: cmake_check_build_system
	$(MAKE) $(MAKESILENT) -f CMakeFiles/Makefile2 GCClassic
.PHONY : GCClassic

# fast build rule for target.
GCClassic/fast:
	$(MAKE) $(MAKESILENT) -f src/GEOS-Chem/Interfaces/GCClassic/CMakeFiles/GCClassic.dir/build.make src/GEOS-Chem/Interfaces/GCClassic/CMakeFiles/GCClassic.dir/build
.PHONY : GCClassic/fast

# Help Target
help:
	@echo "The following are some of the valid targets for this Makefile:"
	@echo "... all (the default if no target is provided)"
	@echo "... clean"
	@echo "... depend"
	@echo "... edit_cache"
	@echo "... rebuild_cache"
	@echo "... APM"
	@echo "... GCClassic"
	@echo "... GeosCore"
	@echo "... GeosRad"
	@echo "... GeosUtil"
	@echo "... GeosUtilHco"
	@echo "... HCO"
	@echo "... HCOI_Shared"
	@echo "... HCOI_Standalone"
	@echo "... HCOX"
	@echo "... Headers"
	@echo "... HeadersHco"
	@echo "... Hg"
	@echo "... History"
	@echo "... Isorropia"
	@echo "... JulDay"
	@echo "... JulDayHco"
	@echo "... KPPFirstPass_fullchem"
	@echo "... KPP_fullchem"
	@echo "... NcdfUtil"
	@echo "... NcdfUtilHco"
	@echo "... ObsPack"
	@echo "... Transport"
	@echo "... gcclassic"
.PHONY : help



#=============================================================================
# Special targets to cleanup operation of make.

# Special rule to run CMake to check the build system integrity.
# No rule that depends on this can have commands that come from listfiles
# because they might be regenerated.
cmake_check_build_system:
	$(CMAKE_COMMAND) -S$(CMAKE_SOURCE_DIR) -B$(CMAKE_BINARY_DIR) --check-build-system CMakeFiles/Makefile.cmake 0
.PHONY : cmake_check_build_system
```

**Comment by msulprizio** [comment]
Hi @ltmurray. After doing `cmake {CodeDir}`, you need to also do `cmake . -DRUNDIR={RunDirPath}` to specify the path to the run directory for the `make install` command. This was added just prior to the 13.0.0-rc.0. Previously, it was assumed that the run directory was one level up (i.e. `-DRUNDIR=..`) but that may not necessarily be the case when compiling from a `build` directory within the code directory or some other location. We can double check the documentation to make sure that is more clear to users.

**Comment by ltmurray** [comment]
Ah, good to know. The README that exists in build does not include that step, contents below.

```
To build GEOS-Chem type:
   cmake ../CodeDir
   make -j
   make install
```

**Comment by msulprizio** [comment]
Thanks for pointing that out. We will correct that.

**Comment by msulprizio** [comment]
The README in the build directory is now updated in commit 49ca6819939660dbacd7e1b0618630f6e1a248b3. This will be included in the final 13.0.0 release.
